burotel/indico_burotel/plugin.py

```diff
@@ -35,19 +50,68 @@ def _add_missing_time(field, data, time_):
         data[field] = dts.dump({field: datetime.combine(date, time_)})[field]
 
 
+def _check_update_permissions(booking):
+    if booking.room.attributes.filter(
```

Comment:
I think this can be simplified to `booking.room.get_attribute_value('electronic-lock') == 'yes'`